hexrd/ui/mask_manager_dialog.py

```diff
@@ -306,3 +308,14 @@ def rebuild_masks(self):
             HexrdConfig().polar_masks_changed.emit()
 
         self.update_masks_list('raw')
+
+    def masks_panel_to_buffer(self):
```

Comment:
Shouldn't this be named `masks_to_panel_buffer`?

Reply:
Oops, yes it should have been, thanks! Fixed the naming